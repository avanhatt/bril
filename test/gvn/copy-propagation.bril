int main(a : int) {
entry:
  b : int = id a;
  c : int = id b;
  d : int = id c;
  ret d;
}