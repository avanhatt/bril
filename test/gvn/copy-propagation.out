int main(a : int) {
entry:
  ret a;
}
